docs/components/image/image.css

```diff
@@ -0,0 +1,33 @@
+.img-res {
+  width: 100%;
+  height: auto;
+  object-fit: cover;
+}
+
+.img-rounded {
+  border-radius: 10px;
+  object-fit: cover;
+}
+
+.img-circle {
+  border-radius: 9999px;
```

Comment:
you can just give border radius 50%

Reply:
The outcome is different if the width and height are not equal, hence I used `9999px`